package/src/components/Attachment/Card.tsx

```diff
@@ -69,13 +69,20 @@ const styles = StyleSheet.create({
 
```

Comment:
@khushal87 can you please a test that covers all cases, testing more of our pure functions is something we need get better at 😁